src/java.desktop/share/classes/sun/print/resources/serviceui_zh_CN.properties

```diff
@@ -29,7 +29,7 @@ border.chromaticity=颜色外观
 border.copies=份数
 border.jobattributes=作业属性
 border.media=介质
-border.output=出纸
```

Comment:
What triggered this ? Just someone noticing it and deciding there's a better translation ?
What does it say ?

Reply:
In the past, it was brought up in review that some of the words were translated to localized words that didn't exactly mean what the original English word meant. I believe Weijun Wang caught some of these in Chinese files in the past.  I'm assuming these are cases where the go team is improving the translation, but I also can't exactly tell. We do suggest changes to the go team, so maybe this is evidence of them improving their tool.

Reply:
It was changed from "paper output" to just "output". Since the original English doesn't specify anything about paper, I guess the new translation is more general and fitting in this case.

The original inclusion of the border.output field was actually part of the output bin update on [JDK-8314070](https://bugs.openjdk.org/browse/JDK-8314070) so it was actually not initially translated by the translation team.